Tests/Integration/IntegrationTestsCommon.psm1

```diff
@@ -0,0 +1,51 @@
+﻿<#
+    .SYNOPSIS
+    Tests if Hyper-V is installed on this OS.
+
+    .OUTPUTS
+    True if Hyper-V is installed. False otherwise.
+#>
+function Test-HyperVInstalled
+{
+    [CmdletBinding()]
+    [OutputType([String])]
+    param
+    (
+    )
+
+    # Ensure that the tests can be performed on this computer
+    $ProductType = (Get-CimInstance Win32_OperatingSystem).ProductType
+    switch ($ProductType) {
+        1
+        {
+            # Desktop OS
+            $HyperVInstalled = (((Get-WindowsOptionalFeature `
+                    -FeatureName Microsoft-Hyper-V `
+                    -Online).State -eq 'Enabled') -and `
+                ((Get-WindowsOptionalFeature `
+                    -FeatureName Microsoft-Hyper-V-Management-PowerShell `
+                    -Online).State -eq 'Enabled'))
+            Break
+        }
+        3
```

Comment:
I think Nano returns 3, but uses Get-WindowsOptionalFeature instead of Get-WindowsFeature. Probably not an issue yet - unless we're thinking of targeting Nano any time soon?